src/AchForm.js

```diff
@@ -105,6 +109,25 @@ export default class AchForm extends Component {
 
     /** tells the component to create an account with the instrument */
     createAccount: PropTypes.bool,
+
+    /**
+     * overrideProps is an object where keys names are ids of the particular 
+     * element in the DOM. `<div id="bank-name" > = "bank-name": {}`. 
+     * Only allowed properties are passed see 
```

Comment:
"Only allowed properties are passed see"...  documentation for full list of allowed properties?